Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.yml

```diff
@@ -31,16 +31,11 @@ name: DUO Admin
 script:
   commands:
   - arguments:
-    - default: false
-      description: The user associated with the logs.
-      isArray: false
+    - description: The user associated with the logs.
```

Comment:
Why did you remove the all `isArray` `secret` nad `default` keys?
Please try to run `demisto-sdk format -i Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.yml`